Source/Core/Common/StringUtil.h

```diff
@@ -201,13 +201,13 @@ std::string PathToFileName(std::string_view path);
 void StringPopBackIf(std::string* s, char c);
 size_t StringUTF8CodePointCount(std::string_view str);
 
-std::string CP1252ToUTF8(std::string_view str);
-std::string SHIFTJISToUTF8(std::string_view str);
-std::string UTF8ToSHIFTJIS(std::string_view str);
-std::string WStringToUTF8(std::wstring_view str);
+std::string CP1252ToUTF8(std::string_view);
+std::string SHIFTJISToUTF8(std::string_view);
+std::string UTF8ToSHIFTJIS(std::string_view);
+std::string WStringToUTF8(std::wstring_view);
 std::string UTF16BEToUTF8(const char16_t* str, size_t max_size);  // Stops at \0
```

Comment:
Why was the `str` name not removed for this one and for StringUTF8CodePointCount when it was removed for other functions?

Reply:
I changed the others to silence these clang-tidy messages:
>Function 'WStringToUTF8' has a definition with different parameter names clang-tidy [readability-inconsistent-declaration-parameter-name](https://clang.llvm.org/extra/clang-tidy/checks/readability/inconsistent-declaration-parameter-name.html)

Most of them name the parameter `input` in the definition and since the purpose of the parameter is obvious for those functions I just removed it entirely in the declarations.

`UTF16BEToUTF8` declaration/definition did not have differing parameter names, and the purpose of the two parameters is perhaps a bit less obvious so I left it as is.

Reply:
Hmm... Usually we put the same name in both locations, but I guess removing the name here is okay too. I'll defer to your choice here.

Reply:
I've added the names back, now matching the name at the definition.